File: server/php/OETableGames/Entity/Player.php
<?php
namespace Entity;


use Core\BaseEntity;

class Player extends BaseEntity
{

	var $PRIMARY_KEY = 'id';

	/**
	 *
	 * 通行证ID
	 *
	 * @var int
	 */
	var $id;
	
	/**
	 *
	 * 昵称
	 *
	 * @var string
	 */
	var $nickname;
	
	/**
	 *
	 * 头像
	 *
	 * @var string
	 */
	var $avatar;
	
	/**
	 *
	 * 性别
	 *
	 * @var int
	 */
	var $gender;
	
	/**
	 *
	 * 房卡
	 *
	 * @var int
	 */
	var $cards;
	
	/**
	 *
	 * 房间ID
	 *
	 * @var int
	 */
	var $roomId;
	
	/**
	 *
	 * 游戏id
	 *
	 * @var int
	 */
	var $gameId;
	
	/**
	 *
	 * 
	 *
	 * @var int
	 */
	var $matchId;
	
	/**
	 *
	 * 积分
	 *
	 * @var int
	 */
	var $score;
	
	/**
	 *
	 * 状态
	 *
	 * @var int
	 */
	var $status;
	
	/**
	 *
	 * 在线时间
	 *
	 * @var bigint
	 */
	var $online;
	
	/**
	 *
	 * 在线IP
	 *
	 * @var bigint
	 */
	var $ipaddr;
	
	/**
	 *
	 * 宝箱钥匙
	 *
	 * @var int
	 */
	var $box_key;

    /**
     *
     * 金币
     *
     * @var int
     */
	var $gold;
}
?>

File: server/java/OETableGames/oetablegames-parent/oetablegames-service/oetablegames-service-player/src/main/java/com/oegame/tablegames/service/player/Player.java
package com.oegame.tablegames.service.player;

import java.io.Serializable;

public class Player implements Serializable
{

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	public long playerId;
	public String nickname = "nickname";
	public String avatar = "avatar";
	public byte gender;
	public int gameId;
	public int roomId;
	public int gold;
	public int cards;
	public long online = 0;
	public boolean isRobot;
	
	public Player()
	{
		
	}

	public Player(long playerId, String nickname, String avatar, byte gender, int roomId, long online, int cards, int gameId, int gold)
	{
		this.playerId = playerId;
		this.nickname = nickname;
		this.avatar = avatar;
		this.gender = gender;
		this.roomId = roomId;
		this.online = online;
		this.cards = cards;
		this.gameId = gameId;
		this.gold = gold;
	}
}
